packages-native/actions/src/camera/TakePictureAdvanced.ts

```diff
@@ -0,0 +1,213 @@
+// This file was generated by Mendix Modeler.
+//
+// WARNING: Only the following code will be retained when actions are regenerated:
+// - the code between BEGIN USER CODE and END USER CODE
+// Other code you write will be lost the next time you deploy the project.
+
+import { Alert, Linking } from "react-native";
+import ImagePicker from "react-native-image-picker";
+
+type PictureSource = "camera" | "imageLibrary" | "either";
+
+type PictureQuality = "original" | "low" | "medium" | "high" | "custom";
+
+/**
+ * @param {MxObject} picture - This field is required.
+ * @param {"NativeMobileResources.PictureSource.camera"|"NativeMobileResources.PictureSource.imageLibrary"|"NativeMobileResources.PictureSource.either"} pictureSource - Select a picture from the library or the camera. The default is to let the user decide.
+ * @param {"NativeMobileResources.PictureQuality.original"|"NativeMobileResources.PictureQuality.low"|"NativeMobileResources.PictureQuality.medium"|"NativeMobileResources.PictureQuality.high"|"NativeMobileResources.PictureQuality.custom"} pictureQuality - Set to empty to use default value 'medium'.
+ * @param {Big} maximumWidth - The picture will be scaled to this maximum pixel width, while maintaing the aspect ratio.
+ * @param {Big} maximumHeight - The picture will be scaled to this maximum pixel height, while maintaing the aspect ratio.
+ * @param {MxObject} pictureData - Additional info about the picture will be stored in this object. Create it before calling this action.
+ * @returns {Promise.<boolean>}
+ */
+export async function TakePicture(
```

Comment:
The name should not be the same as previous JS Action

Reply:
That's the pain of not being able to build locally. I changed it.

Reply:
Why exactly you are not able to run it locally? We have automated tests using Windows machine in Travis. SO you should be able to run on windows.

Reply:
npm install fails because it wants to run a rm command, which the Dell does not understand. I created issue #201 for that.